### PR TITLE
Counts modal update

### PR DESCRIPTION
**Pull request by JulianWielga**

### CODE REVIEW

File: ui/client/components/modals/CalculateCounts/CalculateCountsDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import {useDispatch, useSelector} from "react-redux"
+import {fetchAndDisplayProcessCounts} from "../../../actions/nk"
+import {getProcessId} from "../../../reducers/selectors/graph"
+import "../../../stylesheets/visualization.styl"
+import Dialogs from "../Dialogs"
+import GenericModalDialog from "../GenericModalDialog"
+import {Picker} from "./Picker"
+import {CountsRanges} from "./CountsRanges"
+
+interface State {
```
Comment (lciolecki) [inline ui/client/components/modals/CalculateCounts/CalculateCountsDialog.tsx:14]: Maybe `CounterState`?

File: ui/client/components/modals/CalculateCounts/CountsRangesButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            className="predefinedRangeButton"
+            style={{
+              flex: 1,
+            }}
+            wrapperStyle={{
+              display: "flex",
+              flex: 2,
+            }}
+          >
+            Select more...
```
Comment (lciolecki) [inline ui/client/components/modals/CalculateCounts/CountsRangesButtons.tsx:58]: Missing translation ;)

File: ui/client/http/HttpService.ts [hunk 3]
```diff
@@ -160,6 +160,13 @@ class HttpService {
       .catch(error => this.addError("Cannot fetch status", error))
   }
 
+  fetchProcessesDeployments(processId: string) {
+    return api.get<{performedAt: string, action: "UNARCHIVE" | "ARCHIVE" | "CANCEL" | "DEPLOY"}[]>(`/processes/${processId}/deployments`)
```
Comment (lciolecki) [inline ui/client/http/HttpService.ts:164]: Action can be represented as Enum.

**Review by lciolecki — APPROVED** [review]
LGTM, but please look at comments below.

File: ui/client/http/HttpService.ts (hunk 3 above)
Comment (JulianWielga) [reply ui/client/http/HttpService.ts:164]: it's temporary, it will change soon, makes no sense to change it now

**Pull request merged**